src/amber/environment/settings.cr

```diff
@@ -10,6 +10,27 @@ module Amber::Environment
       skip: Array(String?),
       context: Array(String?))
 
+
+    struct SmtpSettings
+      property host = "127.0.0.1"
+      property port = 1025
+      property enabled = false
+      property username = ""
+      property password = ""
+      property tls = false
+
+      def self.from_hash(settings = {} of String => String | Int32 | Bool | Nil) : self
```

Comment:
This is used enough, I think it should have an alias:
`alias SettingValue = String | Int32 | Bool | Nil`

Reply:
good idea. Since aliases are namespaced, `Value` is `Settings::Value` so that seems good enough, right?